internal/plugin/plugin_helper.go

```diff
@@ -85,3 +85,7 @@ func isPluginInList(plugin transform.Plugin, list []string) bool {
 	}
 	return false
 }
+
+func IsExecAny(mode os.FileMode) bool {
```

Comment:
This is interesting, if we are passing it through, why don't we make it public? 

@sseago

Reply:
No need as of now, updated the plugin package for now and added helper functions there.